apps/main/src/lend/components/ChartOhlcWrapper/index.tsx

```diff
@@ -362,36 +341,38 @@ const ChartOhlcWrapper = ({ rChainId, userActiveKey, rOwmId }: ChartOhlcWrapperP
     )
   }
 
+  const ChartWrapperProps: ChartWrapperProps = {
+    chartType: 'crvusd',
+    chartStatus: currentChart.fetchStatus,
+    chartHeight: chartHeight,
```

Comment:
Not breaking but could use shorthand notation like `chartHeight,` instead of `chartHeight: chartHeight,`. This goes for a couple of properties.